lambdas/catalog-updater/index.js

```diff
@@ -63,17 +63,30 @@ function usagePlanToCatalogObject(usagePlan, swaggerFiles) {
   let catalogObject = {
     id: usagePlan.id,
     name: usagePlan.name,
-    //TO-DO: Allow for customizing image?
-    apis: _.map(usagePlan.apiStages, (apiStage) => ({ id: usagePlan.apiId, image: '/sam-logo.png' }))
+    apis: []
   }
 
-  catalogObject.apis = _.map(catalogObject.apis, (api) => {
+  _.forEach(usagePlan.apiStages, (apiStage) => {
+    let api = {}
 
-    api.swagger = _.filter(swaggerFiles, () => {
-      return swaggerFiles.indexOf(api.id)
-    })
-
-    return api
+    _.chain(swaggerFiles)
```

Comment:
On the use of `chain`, [an optimization to consider](https://medium.com/making-internets/why-using-chain-is-a-mistake-9bc1f80d51ba)